endoreg_db/serializers/patient_finding/patient_finding_classification.py

```
@@ -0,0 +1,39 @@
+from endoreg_db.models import Finding, PatientFindingClassification
```

Ruff (0.12.2)

`endoreg_db.models.Finding` imported but unused

Remove unused import: `endoreg_db.models.Finding`

(F401)